newton/_src/sim/model.py

```
@@ -849,31 +839,29 @@ def collide(
         if collision_pipeline is not None:
             self._collision_pipeline = collision_pipeline
         elif not hasattr(self, "_collision_pipeline"):
-            self._collision_pipeline = CollisionPipeline.from_model(
-                model=self,
-                rigid_contact_max_per_pair=rigid_contact_max_per_pair,
-                soft_contact_max=soft_contact_max,
-                soft_contact_margin=soft_contact_margin,
-                edge_sdf_iter=edge_sdf_iter,
-                requires_grad=requires_grad,
-            )
+            from .collide_unified import BroadPhaseMode, CollisionPipelineUnified  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)